ubg/src/mafia/MafiaNight.js

```diff
@@ -112,20 +112,30 @@ function MafiaNight({userUid, usersData, room,
    */
   function mafiaVote() {
     if (mafiaDecision.length !== 0 && mafiaDecision.length === mafiaTotal) {
+      const today = new Date();
+      const hours = today.getUTCHours();
+      const minutes = today.getUTCMinutes();
       const killed = mafiaDecision[0].vote.uid;
       for (let i = 1; i < mafiaTotal; i++) {
         if (mafiaDecision[i].vote.uid !== killed) {
           setMessage('All mafia must choose the same person to die! ' +
           'Please vote again');
           room.update({
             mafiaDecision: [],
+            mafiaChat: firebase.firestore.FieldValue.arrayUnion(
+                {text: 'Mafia please vote again', hours, minutes},
+            ),
           });
           setChose(false);
           return;
         }
       }
       room.update({
         mafiaKill: mafiaDecision[0].vote,
+        mafiaChat: firebase.firestore.FieldValue.arrayUnion(
```

Comment:
Is this supposed to be in the mafiaChat or the normal chat? And should we check first if the doctor saves that person? Maybe say 'You agreed on NAME' or idk

Reply:
Sure, I can change it to 'You agreed on'. I just want mafias to see who they killed in the end.